src/pal/inc/pal_endian.h

```diff
@@ -98,8 +98,9 @@ inline void SwapGuid(GUID *pGuid)
 #define VALPTR(x) VAL32(x)
 #endif
 
-#if defined(ALIGN_ACCESS) && !defined(_MSC_VER)
-
+#if defined(_TARGET_ARM_) && !defined(_MSC_VER)
+#define LOG2_PTRSIZE	2
```

Comment:
@jkotas , @janvorli  The line 102-103  is derived from ./src/inc/stdmacros.h#L131.  Could you review the lines are correct?